common/processors/concatenate_all_pipelines.py

```diff
@@ -0,0 +1,116 @@
+"""This processor concatenates all datasets into one."""
```

Comment:
oops that came back with the merge conflict resolution. It should have disappeared. my bad.